deploy/debezium-connector.yml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kafka-connectors
+objects:
+- apiVersion: kafka.strimzi.io/v1beta2
+  kind: KafkaConnector
+  metadata:
+    name: ${CONNECTOR_NAME}
+    labels:
+      strimzi.io/cluster: ${KAFKA_CONNECT_INSTANCE}
+  spec:
+    class: io.debezium.connector.postgresql.PostgresConnector
+    tasksMax: ${MAX_TASKS}
+    config:
+      database.server.name: ${DB_SERVERNAME}
+      database.dbname: ${DB_NAME}
+      database.hostname: ${DB_HOSTNAME}
+      database.port: ${DB_PORT}
+      database.user: ${DB_USER}
+      database.password: ${DB_PASSWORD}
+      topic.prefix: ${TOPIC_PREFIX}
+      table.whitelist: ${TABLE_LIST}
+      transforms: outbox
+      transforms.outbox.type: io.debezium.transforms.outbox.EventRouter
+      transforms.outbox.table.field.payload: ${PAYLOAD_NAME}
+      plugin.name: pgoutput
```

Comment:
NOTE: @mjholder debezium generates also event when record is deleted - but there is no action for this event in connector - 

it looks it is possible to disable it by
https://debezium.io/documentation/reference/stable/transformations/applying-transformations-selectively.html#ignoring-tombstone-events

I am not sure about if we should do it regard this comment:

> By default, the tombstones.on.delete property for a connector is set to true so that the connector generates a tombstone after each delete event. If you set the property to false to prevent Debezium from saving tombstone records to Kafka topics, the absence of tombstone records might lead to unintended consequences. Kafka relies on tombstone during log compaction to remove records that are related to a deleted key.

or it can be ignored in connector.

Reply:
we can add it later if we will need it here.